tests/robot-tests/tests/admin_and_public/bau/amend-populated-release-delete-subject-files.robot

```diff
@@ -0,0 +1,163 @@
+*** Settings ***
+Resource  ../../libs/admin-common.robot
+Library  ../../libs/admin_api.py
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       user signs in as bau1
+Suite Teardown    user closes the browser
+
+*** Variables ***
+${DETAILS_HEADING}  Academic Year 2020/21 (not Live)
```

Comment:
I'd change this variable (and remove `AMEND_RELEASE_HEADING`) to "Academic Year 2020/21" and add hard code the "(not Live)" and "(Live - Latest release)" bits into the test cases itself. So line 26 would become
`user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}  ${DETAILS_HEADING} (not Live)`

Reply:
Removed hard coded values from `${DETAILS_HEADING}`